docs/DOCUMENTATION.md

```diff
@@ -40,7 +40,7 @@ The parameters used in `PaymentContext` are explained below:
 The parameters used in the `PaymentPreferences` are explained below:
 
 - `preferredCountryCode`: the preferred country to use when displaying the providers. If the country is invalid, or does not include any providers, the value will fallback to the user's locale.
-- `paymentUseCase`: dictates the wording to display to the user when sending a payment, to clarify what the payment is used for. For example, for SignUp+, use `PaymentUseCase.SignUpPlus`. For a standard payment, use `PaymentUseCase.Default` which is also the default value
```

Comment:
is there a reason for removing this line?

Reply:
It is not removed. It's modified. But for some reason GitHub shows it as removed then added (look at line below)